var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -139,21 +156,7 @@ class Cmake(Package):
     depends_on("gmake", when="platform=darwin")
     depends_on("gmake", when="platform=freebsd")
 
-    # We default ownlibs to true because it greatly speeds up the CMake
-    # build, and CMake is built frequently. Also, CMake is almost always
-    # a build dependency, and its libs will not interfere with others in
-    # the build.
-    variant("ownlibs", default=True, description="Use CMake-provided third-party libraries")
-    variant(
-        "doc",
-        default=False,
-        description="Enables the generation of html and man page documentation",
-    )
-    variant(
-        "ncurses",
-        default=sys.platform != "win32",
-        description="Enables the build of the ncurses gui",
-    )
+    depends_on("qt", when="+qtgui")
```

Comment:
This was removed in https://github.com/spack/spack/pull/39360 for a good reason. Why reintroduce the circular dependency?